build.linux.md

```diff
@@ -0,0 +1,20 @@
+# Build Instructions
+
```

Comment:
May be worth noting that the official system used to compile QC is with `flibitBuild`:

https://icculus.org/finger/flibitijibibo